saleor/graphql/warehouse/mutations.py

```diff
@@ -0,0 +1,84 @@
+import graphene
+
+from saleor.account.models import Address
+from saleor.graphql.core.mutations import ModelDeleteMutation, ModelMutation
+from saleor.graphql.core.types.common import WarehouseError
+from saleor.graphql.warehouse.types import WarehouseCreateInput, WarehouseUpdateInput
+from saleor.warehouse import models
+
+ADDRESS_FIELDS = [
+    "street_address_1",
+    "street_address_2",
+    "city",
+    "city_area",
+    "postal_code",
+    "country",
+    "country_area",
+    "phone",
+]
+
+
+class WarehouseCreate(ModelMutation):
```

Comment:
It would be good to extend the Warehouse model by `Meta`.

Reply:
I would hold on with that since I'd like to first think about refactoring our meta mutations to make them more generic.